style/preview.less

```diff
@@ -309,8 +309,9 @@
 
 	&.expanded {
 		.wikipediapreview-body {
-			max-height: 496px;
-			overflow: hidden;
+			// calculate the max height in onExpand()
+			// max-height: 496px;
```

Comment:
Did you mean to leave this `max-height` commented out?

Reply:
yes, I can remove that, but it gives a highlight where the height of the expanded came from which is useful for debug.

Reply:
Yea figured you were thinking something along that line, I'm fine with leaving it like that